### PR TITLE
Support INC1.9

### PR DESCRIPTION
**Pull request by PenghuiCheng**
# What does this PR do?
Support Intel Neural Compressor 1.9 version.

### CODE REVIEW

**Review by xin3he — APPROVED** [review]

File: optimum/intel/neural_compressor/quantization.py [hunk 1]
```diff
@@ -36,11 +37,11 @@
     WEIGHTS_NAME,
     IncDataLoader,
     _cfgs_to_fx_cfgs,
-    _get_quantizable_ops_recursively,
 )
```
Comment (echarlaix) [inline optimum/intel/neural_compressor/quantization.py:40]: `from optimum.intel.neural_compressor.utils import WEIGHTS_NAME, IncDataLoader, _cfgs_to_fx_cfgs` will allow to pass the check_code_quality test

**Comment by echarlaix** [comment]
Thanks for the PR, it looks good to me. Could you merge the branch `main` in `penghuic/Support_INC1.9` in order to have the results of the INC tests as well? (that a recent change inadvertently disabled)

File: optimum/intel/neural_compressor/quantization.py (hunk 1 above)
Comment (PenghuiCheng) [reply optimum/intel/neural_compressor/quantization.py]: done

**Comment by PenghuiCheng** [comment]
> Thanks for the PR, it looks good to me. Could you merge the branch `main` in `penghuic/Support_INC1.9` in order to have the results of the INC tests as well? (that a recent change inadvertently disabled)

rebased on main branch

**Comment by echarlaix** [comment]
Good to merge on my side, please confirm that the recent additions seems fine to you @PenghuiCheng

**Pull request merged**